models/maat/R/model2netcdf.MAAT.R

```diff
@@ -12,7 +12,8 @@
 ##' Function to convert MAAT model output to standard netCDF format
 ##'
 ##' Converts all output contained in a folder to netCDF.
-##'
+##' @name model2netcdf.MAAT
```

Comment:
`@name` tag is redundant with function name. Most style guides recommend against using it (with modern versions of Roxygen).

```suggestion
```